Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md

```diff
@@ -0,0 +1,25 @@
+## GetConfigurationObject request
+
+### Description
+GetConfigurationObject is a request to retrieve a ConfigurationObject from on a device. The attributes with OBIS code 0-1:94.31.3.255 is used to access the GPRS_operation_mode setting and following flags:
```

Comment:
"The attributes with OBIS code ... is" looks not right. It's the configuration object in the electricity meter with the OBIS code that holds a number of attributes as described.